JosephNjuguna/Questioner
# Ft user profile ui 162950811

### #162950811 user can view their profile page
#### What does this PR do?
this pr adds a profile view page 
#### Description of Task to be completed?
enabling  user to:
1. view their profile information
2. view the top questions on the meetup they are scheduled to attend
3. view questions they have asked before on different meetups
4. view questions they have commented on on different meetups
5.view suggestions to other meetups that the user might like

#### How should this be manually tested?
clone tis repo to local machine and navigate following this : `UI/templates/profile.html`
check your internet connection and open this link  `https://josephnjuguna.github.io/Questioner/UI/templates/profile.html`

#### What are the relevant pivotal tracker stories?
`https://www.pivotaltracker.com/n/projects/2235137`  _a user can view their  profile page_

![screenshot from 2019-01-06 16-01-24](https://user-images.githubusercontent.com/44730807/50749616-16761880-1251-11e9-9ca2-93bea4323d31.png)
